client/components/Home.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { render } from 'react-dom';
+
+class Home extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  //functionality for input field and button
+
+    // send this data to our API and potentially get info back  
+    // send a request to /data endpoint with a JSON object 
+    // { address: address}
+    // make a fetch request to the endpoint "/election"
+    // send the  JSON object
+    // use .thens and harness the data that is returned
+    // use the data in some way
+
+  //render method will render header, form, and button and functionality will be added as attirutes in renderings
+  render() {
+    return (
+      <div>
+        <header className="header"><h1>Who's On The Ballot?</h1></header>
+        <div id='form'>
+          <form>
```

Comment:
There seems to be a double up on tags - one div with the id form and a form tag. Is the form tag necessary? Let's discuss!

Reply:
We removed the outer div and gave the <form> a class. I think we need to '<form> to connect our input and button into one event

(is it best practice to comment on each of your comments? I have no idea)